File: Payanak/Original/ClientApp/src/app/shared/model/user.model.ts
import {AccountInfoModel} from './account-info.model';
import {PersonalInfoModel} from './personal-info.model';
import {AdditionalInfoModel} from './additional-info.model';
import {AddressInfoModel} from './address-info.model';

export class UserModel {
  accountInfo: AccountInfoModel;
  additionalInfo: AdditionalInfoModel;
  personalInfo: PersonalInfoModel;
  addressInfo: AddressInfoModel;
  roles: number[] = [];
}

File: Payanak/Original/ClientModels/UserModel.cs
using System.Collections.Generic;

namespace Backend.ClientModels
{
    public class UserModel
    {
        public AccountInfoModel AccountInfo { get; set; }
        public AdditionalInfoModel AdditionalInfo { get; set; }
        public PersonalInfoModel PersonalInfo { get; set; }
        public AddressInfoModel AddressInfo { get; set; }
        public List<long> Roles { get; set; } = new List<long>();
    }
}